### PR TITLE
Create project model

### LINKED ISSUE #16: Create Table: project
### Overview
We need to create the Project table so that we can update a shared data store across hackforla.org, vrms, civictechjobs, and tables (onboarding) project.

#### Details
A table and a model are the same thing

### Action Items
- [ ] identify if table has a description (see spreadsheet under Resources)
  - [ ] if not, reach out to PD leads
- [ ] identify and document  (below) what other tables are associated (see ERD under Resources)
- [ ] create a single model in Django (defining schema)
- [ ] Write a test for the relationships this model will have with other models (e.g., creating a user and assigning them a set of permissions on a project).
- [ ] Write an API end point
- [ ] write API unit tests
- [ ] Document the endpoint in Swagger

### Resources/Instructions
- See [People Depot Resources wiki page](https://github.com/hackforla/peopledepot/wiki/Resources-and-Links) for links

#### Description
-

#### Associated Tables
-

#### Swagger Endpoint Link
-

### PR DESCRIPTION
**Pull request by fyliu**
- closes #16

- Added project model
- Added project API endpoints
- Added simple model test

### CODE REVIEW

File: app/core/models.py [hunk 1]
```diff
@@ -1,5 +1,150 @@
...
+    # current_status_id = models.ForeignKey("status", on_delete=models.PROTECT)
+    hide = models.BooleanField(default=True)
+    # location_id = models.ForeignKey("location", on_delete=models.PROTECT)
+    slack_url = models.URLField(blank=True)
+    google_drive_url = models.URLField(blank=True)
+    google_drive_id = models.CharField(max_length=255, blank=True)
+    hfla_website_url = models.URLField(blank=True)
+    # leads = models.ManyToManyField("lead")
+    # leadership_type_id = models.ForeignKey("leadership_type", on_delete=models.PROTECT)
+    image_logo = models.URLField(blank=True)
```
Comment (cnk) [inline app/core/models.py:143]: Should the image fields be [ImageFields](https://docs.djangoproject.com/en/4.1/ref/models/fields/#django.db.models.ImageField) so Django can manage them? You would need to sort out how you will be storing the images - probably an S3 bucket for production but a local media volume for dev. You can use django-storages and a different MEDIA_URL configurations for dev and prod.

**Review by cnk — CHANGES_REQUESTED** [review]
Again, ok to merge but I would like to know your thoughts on my suggestions.

File: app/core/models.py [hunk 2]
```diff
@@ -1,5 +1,150 @@
...
+"Authorization: token [gh_PAT]" https://api.github.com/orgs/[org]',
+    )
+    github_primary_repo_id = models.CharField(
+        max_length=9,
+        blank=True,
+        help_text='Can be retrieved from gh api with the following: curl -H \
+"Authorization: token [gh_PAT]" \
+https://api.github.com/repos/[org]/[repo]',
+    )
+    github_primary_url = models.CharField(max_length=255, blank=True)
```
[the remark this answers is not in the document]
Comment (fyliu) [reply app/core/models.py:133]: Another great catch! This one I should have seen.

File: app/core/models.py (hunk 1 above)
Comment (fyliu) [reply app/core/models.py:143]: This makes sense.
- Added #91 for discussion

**Pull request merged**